stm32f4/uvisor/debug/stm32f4_memory_map.c

```diff
@@ -134,18 +134,17 @@ static const MemMap g_mem_map[] = {
     {"TPI",     TPI_BASE,               TPI_BASE               + sizeof(TPI_Type)             },
 };
 
-const MemMap* memory_map_name(uint32_t addr)
+const MemMap *memory_map_name(uint32_t addr)
 {
     int i;
     const MemMap *map;
 
     /* find system memory region */
     map = g_mem_map;
-    for(i = 0; i < UVISOR_ARRAY_COUNT(g_mem_map); i++)
-        if((addr >= map->base) && (addr <= map->end))
+    for (i = 0; i < UVISOR_ARRAY_COUNT(g_mem_map); i++)
+        if ((addr >= map->base) && (addr <= map->end))
             return map;
-        else
-            map++;
+        map++;
```

Comment:
I think here the functionality is changed: The `map++` statement is executed outside the `for` loop now that the `else` has been removed. It did not require brackets before, now it does.